File: Python/Q20.py
# Q20.py - Program to Display Fibonacci Sequence
n = int(input("Enter number of terms: "))
a, b = 0, 1
print(a, b, end=' ')
for _ in range(n-2):
    a, b = b, a+b
    print(b, end=' ')
print()

File: Java/Q20.java
import java.util.*;

public class FibonacciSequence {

    public static void main(String[] args) {
        int n = 10; // Number of terms to display
        int firstTerm = 0;
        int secondTerm = 1;

        System.out.println("Fibonacci Series up to " + n + " terms:");

        for (int i = 1; i <= n; ++i) {
            System.out.print(firstTerm + ", ");

            // Compute the next term
            int nextTerm = firstTerm + secondTerm;
            firstTerm = secondTerm;
            secondTerm = nextTerm;
        }
    }
}
